extensions/arc/package.json

```diff
@@ -339,6 +340,17 @@
                             "value": "Direct"
                           }
                         },
+                        {
+                          "type": "text",
```

Comment:
Can you move cluster name before custom location?

Reply:
Done
![image](https://user-images.githubusercontent.com/22386690/176334105-c5986b24-779a-421d-bff5-71fccf97fcd5.png)